client/src/modules/stock/lots/modals/schedule.modal.html

```diff
@@ -0,0 +1,88 @@
+<form
+  name="ModalForm"
+  data-modal="stock-lots-schedule"
+  ondragstart="return false;" ondrop="return false;"
+  novalidate>
+
+  <div class="modal-header">
+    <h2 translate>LOTS_SCHEDULE.TITLE</h2>
```

Comment:
To make this more uniform, it would be nice to use the `.headercrumb` class that we generally use in our modals.  It would also give you more space to work with in the modal itself.

Reply:
Done.